init.py

```diff
@@ -43,28 +43,18 @@ def replace(filepath):
         with open(filepath, "w", encoding="utf-8") as hnd:
             hnd.write(content)
 
-    files_to_replace = ["LICENSE"]
-    directories_to_replace = [os.path.join("src", "fillname"), "tests", "doc"]
-    for filename in files_to_replace:
-        filepath = os.path.join(filename)
-        replace(filepath)
+    dirs = [os.path.join("src", "fillname"), "tests", "doc"]
+    files = ["setup.cfg", "noxfile.py", "README.md", "LICENSE", ".pre-commit-config.yaml", ".coveragerc"]
 
-    for rootpath in directories_to_replace:
+    for rootpath in dirs:
         for dirpath, _, filenames in os.walk(rootpath):
             for filename in filenames:
-                if not filename.endswith(".py"):
+                if not filename.endswith(".py") and not filename.endswith(".md"):
```

Comment:
@rkaminsk  I did use fillname in the documentation md (for the install and use). Is there a reason why we are skipping them?

Reply:
We don't. Only other files are skipped.